tracer_test.go

```diff
@@ -597,17 +597,11 @@ func TestTracerDefaultTransport(t *testing.T) {
 		require.Error(t, err)
 		assert.EqualError(t, err, "failed to parse ELASTIC_APM_SERVER_TIMEOUT: invalid duration never")
 
-		// Implicitly created Tracers will have a discard tracer.
+
```

Comment:
I'd recommend running the go linter locally. This additional new line is not necessary.

Reply:
You will also want to run the tests locally, as they currently are failing.

Reply:
`make test` did fail consistently in CI, as well as locally here.
This was because the tracecontext example relies on a discarded tracer. But now, it had an inactive one. I've fixed it.